.storybook/main.js

```diff
@@ -40,5 +42,14 @@ function transpileModules(config) {
     }
     return rule;
   });
+  // Expose environment variables to Storybook
+  config.plugins = [
+    ...config.plugins,
+    new webpack.DefinePlugin({
+      'process.env.UNSAFE_DISABLE_ACCESSIBILITY_ERRORS': JSON.stringify(
```

Comment:
Note: here I had to use `JSON.stringify()` as mentioned in the [plugin's docs](https://webpack.js.org/plugins/define-plugin/), but in the dashboard's Next.js config I didn't need to (doing so resulted in `UNSAFE_DISABLE_ACCESSIBILITY_ERRORS === '"true"'`), I assume Next.js is doing it under the hood